### PR TITLE
file structure #28

### PR DESCRIPTION
**Pull request by safaaamro**
- [ ] Make file structure for a project.
File Structure:
 - client
   - public
     - index.html
   - src
     - Components
        - Home
          - index.js
          - home.sass
        - Header
          - index.js
          - header.sass
       - Brands
          - index.js
          - brands.sass
       - Brand
          - index.js
          - brand.sass
       - Feedback
          - index.js
          - feedback.sass
    - package.json
    - package-lock.json
    - .gitignore
- server
   - server.js
   - app.js
   - controllers
- package.json
 - pckage-lock.json
 - .gitignore
 - .eslintrc.json
 - .env

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,6 +24,8 @@
     "concurrently": "^4.1.0",
     "env2": "^2.2.2",
     "express": "^4.16.4",
-    "i": "^0.3.6"
+    "i": "^0.3.6",
```
Comment (shahenazmonia) [inline package.json]: why did you install this module?

File: package.json [hunk 2]
```diff
@@ -22,6 +24,8 @@
     "concurrently": "^4.1.0",
```
Comment (shahenazmonia) [inline package.json]: ```concurrently``` module should be in devDependancies because you don't need it in the production mode

File: package.json [hunk 3]
```diff
@@ -22,6 +24,8 @@
...
     "env2": "^2.2.2",
     "express": "^4.16.4",
-    "i": "^0.3.6"
+    "i": "^0.3.6",
+    "nodemon": "^1.18.10",
```
Comment (shahenazmonia) [inline package.json]: ```nodemon``` module should be in devDependancies because you don't need it in the production mode

File: package.json (hunk 1 above)
Comment (Samaamro20) [reply package.json]: Done

File: package.json (hunk 2 above)
Comment (Samaamro20) [reply package.json]: Done

File: package.json (hunk 3 above)
Comment (Samaamro20) [reply package.json]: Done

**Pull request merged**